File: fizzBuzz.js
// 1. FizzBuzz
// Mostre números de 1–100:
// múltiplo de 3 → Fizz
// múltiplo de 5 → Buzz
// ambos → FizzBuzz

for (let i = 1; i <= 100; i++){
    if (i % 3 === 0 && i % 5 === 0) {
        console.log(`${i} -> FizzBuzz`)
    } else if (i % 3 === 0) {
        console.log(`${i} -> Fizz`)
    } else if (i % 5 === 0) {
        console.log(`${i} -> Buzz`)
    } else { console.log (`${i}`)}
}

File: python/fizz_buzz.py
# 1. FizzBuzz
# Mostre números de 1–100:
# múltiplo de 3 → Fizz
# múltiplo de 5 → Buzz
# ambos → FizzBuzz

for num in range(1, 101):
    if ((num % 3 == 0) and (num % 5 == 0)):
        print(f"{num} -> Fizz Buzz")
    elif(num % 3 == 0):
        print(f"{num} -> Fizz")
    elif(num % 5 == 0):
        print(f"{num} -> Buzz")
    else:
        print(num)
